diff --git a/node_modules/@google/generative-ai/dist/generative-ai.d.ts b/node_modules/@google/generative-ai/dist/generative-ai.d.ts
index 5b68f7b..6ce5560 100644
--- a/node_modules/@google/generative-ai/dist/generative-ai.d.ts
+++ b/node_modules/@google/generative-ai/dist/generative-ai.d.ts
@@ -632,7 +632,11 @@ export declare enum HarmCategory {
     HARM_CATEGORY_HATE_SPEECH = "HARM_CATEGORY_HATE_SPEECH",
     HARM_CATEGORY_SEXUALLY_EXPLICIT = "HARM_CATEGORY_SEXUALLY_EXPLICIT",
     HARM_CATEGORY_HARASSMENT = "HARM_CATEGORY_HARASSMENT",
-    HARM_CATEGORY_DANGEROUS_CONTENT = "HARM_CATEGORY_DANGEROUS_CONTENT"
+    HARM_CATEGORY_DANGEROUS_CONTENT = "HARM_CATEGORY_DANGEROUS_CONTENT",
+
+    //Added by @samrahimi, only supported by gemini-1.5-pro-exp-0827, gemini-1.5-flash-exp-0827, gemini-1.5-flash-8B-exp-0827 
+    HARM_CATEGORY_CIVIC_INTEGRITY = "HARM_CATEGORY_CIVIC_INTEGRITY"
+
 }
 
 /**
